### PR TITLE
Feature/dp 739 pdf category connected person data added

### PR DESCRIPTION
**Pull request by JBaigGoaco**
Updated PDF work

image

image

image

### CODE REVIEW

File: Services/CO.CDP.DataSharing.WebApi/PdfGenerator.cs [hunk 1]
```diff
@@ -216,8 +242,42 @@ private void AddConnectedPersonInformationSection(ColumnDescriptor col, List<Con
...
+
+                if (!string.IsNullOrEmpty(identifier.Scheme))
+                    col.Item().Element(container => AddTwoColumnRow(container, "Scheme", identifier.Scheme));
+
+                if (!string.IsNullOrEmpty(identifier.Scheme))
+                    col.Item().Element(container => AddTwoColumnRow(container, "Legal name", identifier.LegalName));
+
+                if (!string.IsNullOrEmpty(identifier.Scheme))
+                    col.Item().Element(container => AddTwoColumnRow(container, "Uri", identifier.Uri));
+            }
```
Comment (shilpigoel-goaco) [inline Services/CO.CDP.DataSharing.WebApi/PdfGenerator.cs:264]: @JBaigGoaco just one question, don't we need to show Primary Identifier?

Comment (JBaigGoaco) [reply Services/CO.CDP.DataSharing.WebApi/PdfGenerator.cs:264]: I think the Primary identifier is the VAT number displayed at the top of the PDF. or maybe i am wrong

Comment (jakzal) [reply Services/CO.CDP.DataSharing.WebApi/PdfGenerator.cs:264]: @JBaigGoaco VAT is never the primary identifier.

Comment (JBaigGoaco) [reply Services/CO.CDP.DataSharing.WebApi/PdfGenerator.cs:264]: image

Yes we are displaying the Primary Identifier.

**Review by dpatel017 — APPROVED** [review]

**Review by jakzal — APPROVED** [review]

**Review by shilpigoel-goaco — APPROVED** [review]

**Pull request merged**